stat/stat_test.go

```diff
@@ -1437,7 +1437,10 @@ func TestQuantile(t *testing.T) {
 	cumulantKinds := []CumulantKind{
 		Empirical,
 		LinInterp,
+		Linear,
 	}
+	const tol = 1e-12
```

Comment:
If there is a `tol`, it should be on the first line of the function, with a blank line between it and the following code.